mslib/msui/autoplot_dockwidget.py

```diff
@@ -27,15 +27,17 @@
     limitations under the License.
 """
 
-import json
 import os
-import click
-from mslib.utils.mssautoplot import main as cli_tool
+import json
+from datetime import datetime
+
 from PyQt5.QtWidgets import QWidget, QFileDialog, QTreeWidgetItem, QMessageBox
 from PyQt5 import QtCore
+import click
+
+from mslib.utils.mssautoplot import cli_tool
```

Comment:
this needs to become changed it is 

from mslib.utils.mssautoplot import main as cli_tool